model/riscv_insts_vext_utils.sail

```diff
@@ -13,16 +13,16 @@ mapping maybe_vmask : string <-> bits(1) = {
 /* Check for valid vsew and lmul values */
 val vcheck_vsew_lmul : (int, real) -> bool
 function vcheck_vsew_lmul(vsew_bits, lmul) = {
-  if  vsew_bits < 8     |
-      vsew_bits > 64    |
-      lmul      < 0.125 |
-      lmul      > 8.0 then false else true
+  if  vsew_bits >= 8     &
+      vsew_bits <= 64    &
+      lmul      >= 0.125 &
+      lmul      <= 8.0   then true else false
```

Comment:
You missed the main point; `if X then true else false` is adding unnecessary complexity, it's entirely equivalent to just `X` since `X` is already a bool (and `if X then false else true` is just `not(X)`.

Reply:
Fixed